scripts/exgdas_atmos_verfrad.sh

```diff
@@ -0,0 +1,222 @@
+#! /usr/bin/env bash
+
+source "$HOMEgfs/ush/preamble.sh"
```

Comment:
Same comment. Can likely ingest the v17 copy here but need to remove the preamble.